tests/misc_env/install_prereq.py

```diff
@@ -173,6 +184,41 @@ def enable_rhel_rpms(ceph, distro_ver):
         )
 
 
+def enable_rhel_eus_rpms(ceph, distro_ver):
+    """
+    Setup cdn repositories for rhel systems
+    Args:
+        distro_ver: distro version details
```

Comment:
"distro version details" does not help me understand what type of argument to pass in here. Would you please provide an example?

Reply:
example added.